src/runtime/pybuffer.cs

```diff
@@ -88,7 +88,7 @@ private static char OrderStyleToChar(BufferOrderStyle order, bool eitherOneValid
         /// </summary>
         public static long SizeFromFormat(string format)
         {
-            if (Runtime.pyversionnumber < 39)
+            if (Runtime.PyVersion.Minor < 9)
```

Comment:
You must compare the full version. E.g. `Runtime.PyVersion < new Version(3, 9)`

Reply:
@lostmsu done